### DISCUSSION TITLE
is there any way to set a svg image as a background image?

### DISCUSSION THREAD

**Discussion by park78951** [vercel/next.js#51639] [Help]
### Summary

i'm importing css in _app.tsx in nextjs project, and there are many images of jpg, png and svg as background image.
All of jpg and png images are clearly shown, but only svg images are not.
Is there any way to figure this out? I'd like to use svg images as background image.

**Comment by icyJoseph** [comment]
Show code on how you are trying to do it. To use an svg as background you need a data format like: `data:image/svg+xml,---rest of the svg` - I think base64 also works, but without seeing what you are trying, I am not sure how to guide/help.
